src/script/interpreter.cpp

```diff
@@ -1943,6 +1943,9 @@ static bool VerifyWitnessProgram(const CScriptWitness& witness, int witversion,
             }
             return set_success(serror);
         }
+    } else if (!is_p2sh && witversion == 1 && program.size() == 2 && program[0] == 0x4e && program[1] == 0x73) {
```

Comment:
nit: will be nice to define the program values `0x4e` and `0x73` for `PayToAnchor` as `const` and reuse them, as the raw values appears in several places.

Reply:
I made another CScript helper function and tried to make things more self-documenting. Let me know what you think.

Reply:
looks much better to me 💯